src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/UpdateSentenceDateService.kt

```diff
@@ -57,19 +60,29 @@ class UpdateSentenceDateService(
       staffMember = staffMember,
     )
 
-    val licenceIsNowInHardStopPeriod = releaseDateService.isInHardStopPeriod(updatedLicenceEntity)
+    val licencePreviouslyInHardStopPeriod = releaseDateService.isInHardStopPeriod(licenceEntity)
+    val licenceCurrentlyInHardStopPeriod = releaseDateService.isInHardStopPeriod(updatedLicenceEntity)
 
-    if (licenceIsNowInHardStopPeriod && updatedLicenceEntity is CrdLicence) {
+    if (licenceCurrentlyInHardStopPeriod && updatedLicenceEntity is CrdLicence) {
```

Comment:
do you need to only do this if they weren't previously in the hard stop period?

Reply:
(this can't work exactly like the time out job as that filters out timed out licences already)